packages/cli/test/fixtures/runtime-resolve/platformatic.json

```diff
@@ -15,25 +15,48 @@
     {
       "id": "piquant-combat",
       "path": "services/piquant-combat",
+      "config": "services/piquant-combat/platformatic.json"
+    },
+    {
+      "id": "without-path",
       "config": "services/piquant-combat/platformatic.json",
-      "useHttp": false
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
+    },
+    {
+      "id": "piquant-existing",
+      "path": "services/piquant-combat",
+      "config": "services/piquant-combat/platformatic.json",
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
+    },
+    {
+      "id": "piquant-resolved",
+      "path": "{MISSING_ENV_VAR}",
+      "config": "services/piquant-combat/platformatic.json",
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
     },
     {
       "id": "external-service-1",
       "config": "platformatic.json",
-      "url": "http://github.com/test-owner/test-app-1.git"
+      "path": "services/external-service-1",
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
     },
     {
       "id": "external-service-2",
       "config": "platformatic.json",
       "path": "{SOME_ENV_VAR}",
-      "url": "http://github.com/test-owner/test-app-2.git"
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
     },
     {
       "id": "external-service-3",
       "config": "platformatic.json",
       "path": "{MISSING_ENV_VAR}",
-      "url": "http://github.com/test-owner/test-app-3.git"
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
+    },
+    {
+      "id": "external-service-4",
+      "config": "platformatic.json",
+      "path": "../non-existent",
+      "url": "https://github.com/platformatic/wattpm-fixtures.git"
```

Comment:
are those actually resolved in the tests? I would prefer if the tests would not use the public network.

Reply:
At the moment they do. Let me find a way not to actually resolve the network.